src/main/cliapp/src/components/Filters/FilterComponentMultiSelect.js

```diff
@@ -57,6 +57,18 @@ export function FilterComponentMultiSelect({ isEnabled, filterConfig, currentFil
 		return (<div>{option.optionLabel}</div>);
 	};
 
+	const dataProviderItemTemplate = (option) => {
```

Comment:
Instead of calling this dataproviderTemplate can we call it uppercase template?

Reply:
@oblodgett  Name has been updated